src/Microsoft.SqlTools.SqlCore/ObjectExplorer/StatelessObjectExplorer.cs

```diff
@@ -38,10 +38,18 @@ public static async Task<TreeNode[]> Expand(string connectionString, SecurityTok
         {
             using (SqlConnection conn = new SqlConnection(connectionString))
             {
-                conn.AccessToken = accessToken?.Token;
-                conn.Open();
+                // In case of MWC access token based connections, we need to open the sql connection first before creating  the server connection
```

Comment:
Is this actually related to MWC? I'd just remove that to avoid confusion for people who don't know what MWC is.